docs/server/source/appendices/firewall-notes.md

```diff
@@ -63,6 +64,11 @@ You may want to have Gunicorn and the reverse proxy running on different servers
 Port 28015 is the default port used by RethinkDB client driver connections (TCP). If your BigchainDB node is just one server, then Port 28015 only needs to listen on localhost, because all the client drivers will be running on localhost. Port 28015 doesn't need to accept inbound traffic from the outside world.
 
 
+## Port 28017
```

Comment:
Oh! I didn't know that was deprecated since version 3.2. I removed all mentions of that port.